client/src/Pages/Uptime/Create/index.jsx

```
@@ -1,137 +1,203 @@
+import Dialog from "../../../Components/Dialog";
+import PulseDot from "../../../Components/Animated/PulseDot";
+import SkeletonLayout from "./skeleton";
 
 // Utils
+import { useParams } from "react-router";
 import { useTheme } from "@emotion/react";
 import { useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
 import { monitorValidation } from "../../../Validation/validation";
 import { createToast } from "../../../Utils/toastUtils";
+import { useTranslation } from "react-i18next";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)